info_terminal/infremen/src/infremen.cpp

```diff
@@ -0,0 +1,243 @@
+#include <stdlib.h>
+#include "ros/ros.h"
+#include <mongodb_store/message_store.h>
+#include <boost/foreach.hpp>
+#include <strands_navigation_msgs/NavStatistics.h>
+#include <strands_navigation_msgs/TopologicalMap.h>
+#include <strands_navigation_msgs/TopologicalNode.h>
+#include <strands_executive_msgs/AddTask.h>
+#include <strands_executive_msgs/SetExecutionStatus.h>
+#include <mongodb_store_msgs/StringPair.h>
+#include <std_msgs/Empty.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <std_srvs/Empty.h>
+#include <std_msgs/String.h>
+#include <std_msgs/Int32.h>
```

Comment:
`std_msgs`, `strands_navigation_msgs`, `strands_executive_msgs`, `mongodb_store_msgs`, `geometry_msgs` are not listed in the package.xml or CMakeLists.txt